naresh/src/main/java/com/lftechnology/phpjava/ems/dao/UserDaoImpl.java

```diff
@@ -0,0 +1,207 @@
+package com.lftechnology.phpjava.ems.dao;
+
+import com.lftechnology.phpjava.ems.entities.User;
+import com.lftechnology.phpjava.ems.utlis.DbFactory;
+import com.lftechnology.phpjava.ems.utlis.PasswordHashGenerator;
+
+import java.sql.*;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * UserDaoImpl
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 08 2016
+ */
+public class UserDaoImpl implements DaoSignature<User> {
+    protected Connection conn = DbFactory.getConnection();
+    protected PreparedStatement stmt = null;
+
+    /**
+     * @param length
+     * @return
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    public static String preparePlaceHolders(int length) {
+        return String.join(",", Collections.nCopies(length, "?"));
+    }
+
+    /**
+     * @param preparedStatement
+     * @param values
+     * @throws SQLException
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    public static void setValues(PreparedStatement preparedStatement, Object... values) throws SQLException {
+        for (int i = 0; i < values.length; i++) {
+            preparedStatement.setObject(i + 1, values[i]);
+        }
+    }
+
+    /**
+     * @return
+     * @throws SQLException
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    @Override
+    public List<User> findAll() throws SQLException {
+        return null;
+    }
+
+    /**
+     * @param user
+     * @return
+     * @throws SQLException
+     * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+     */
+    @Override
+    public int insert(User user) throws SQLException {
+        int lastInsertedId = 0;
+        String sql = "INSERT INTO user (username, password, is_terminated) VALUES ( ?, ? , 0);";
```

Comment:
you don't need to insert the is_terminated value, you have already set the default = false